### DISCUSSION TITLE
Could we have a RC 2.1 release

### DISCUSSION THREAD

**Discussion by TaylorEdm** [OrchardCMS/OrchardCore#9780] [General]
As the title says, could we have a release candidate 2.1?  I would like to see a stable release that has the improvements made over the past year (specifically related to the caching). I know there are breaking changes in the current nightly build but I went through that painful process since the RC 2.0 is so old.

**Comment by Piedone** [comment]
The upcoming release, which will be 1.0, is in the works: https://github.com/OrchardCMS/OrchardCore/issues/7206

**Comment by ns8482e** [comment]
I wish we have rc3 as rc2 on nuget is still on net core 3.1, and having rc3 allows people to use OC with .net 5 via nuget.
